.github/workflows/publish-python.yml

```diff
@@ -0,0 +1,24 @@
+name: publish-sdk-python
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - sdk/python/**
+    paths:
+      - "sdk/python/**"
+      - .github/workflows/publish-python.yml
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-go@v3
+        with:
+          go-version: 1.19
+      - uses: magefile/mage-action@v2
+        with:
+          version: latest
+          args: sdk:python:publish ${{ github.ref_name }}
+        env:
+          PYPI_PAT: ${{ secrets.RELEASE_PYPI_TOKEN }}
```

Comment:
I don't know what this name should be: `RELEASE_PYPI_TOKEN`. Just made it up.

Reply:
`RELEASE_PYPI_TOKEN` SGTM (/cc @gerhard)

Do you need help putting the secret in GHA?